.circleci/config.yml

```diff
@@ -0,0 +1,215 @@
+# iOS CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/ios-migrating-from-1-2/ for more details
+#
+version: 2
+jobs:
+
+  build:
+    working_directory: ~/code
+    docker:
+      - image: circleci/android:api-27-alpha
+    environment:
+      JVM_OPTS: -Xmx3200m
+    steps:
+      - checkout
+      - run:
+          name: build the whole project
+          command: bash gradlew build -x test
+
+  unittest:
+    working_directory: ~/code
+    docker:
+      - image: circleci/android:api-27-alpha
+    resource_class: large
+    environment:
+      JVM_OPTS: -Xmx3200m
+
+    steps:
+      - checkout
+
+      - run:
+          name: run unit tests
+          command: |
+            python3 CircleciScripts/run_unittest.py test_results
+      - run:
+          name : check unit test result 
+          command : bash CircleciScripts/check_test_result.sh test_results
+          
+      - store_artifacts:
+          path: test_results
+
+
+  release_tag:
```

Comment:
When will this step be run?

Reply:
We want to only run build job currently.